### PR TITLE
Added example of async server data

### PR DESCRIPTION
**Pull request by Stanko**
Example of async server data for this issue https://github.com/workco/marvin/issues/22

### CODE REVIEW

File: source/js/server.js [hunk 1]
```diff
@@ -45,6 +43,39 @@ app.use((req, res) => {
   }
 
   // TODO how to handle 50x errors?
+};
+
+const fetchData = (req, res, store) => {
+  const routeMapper = {
+    '/': () => {
```
Comment (Stanko) [inline source/js/server.js]: Please note that this is naive route matching, it won't work if you have params in your route. For cases like this, you need to write a bit smarter route to actions resolver.

Comment (andrewmartin) [reply source/js/server.js]: @Stanko is this the place you'd use `match` and properly grab the right action, if you needed/wanted to?

This is great, by the way! Thanks for sharing.

Comment (Stanko) [reply source/js/server.js]: You are welcome! 🎉 

Yes, that is the place, I just matched `/` and if you start the project you'll see that data is fetched and rendered on the home page.

**Comment by Stanko** [comment]
v1 brought a lot of changes, including async data on the server.

**Pull request closed without merging**